app/src/main/java/io/gnosis/safe/ui/signing/owners/OwnerSelectionViewModel.kt

```diff
@@ -53,7 +55,10 @@ class OwnerSelectionViewModel
     fun importOwner() {
         safeLaunch {
             val key = derivator.keyForIndex(ownerIndex)
-            //TODO: store key
+            ownerKeyHandler.storeKey(key)
```

Comment:
it might be better to have Owner model 
```kotlin
data class Owner(
val address: Solidity.Address,
val key: EncryptedByteArray
)
```
and then have ownerHandler deal with owners and not separated attributes

Reply:
I don't think, we can go with EncryptedByteArray here. At least to store it, we need to receive the unencrypted key.